### PR TITLE
Bugfix : refreshing access token

### PR DESCRIPTION
**Pull request by svetzhelev**
Refreshing the access token when expires_in is nil from the authorization server.

### CODE REVIEW

File: MHIdentityKit.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -771,9 +771,9 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 9954AE591E9D169700F2A73D /* Build configuration list for PBXNativeTarget "MHIdentityKit-iOS" */;
 			buildPhases = (
+				9954AE421E9D169700F2A73D /* Headers */,
```
Comment (KoCMoHaBTa) [inline MHIdentityKit.xcodeproj/project.pbxproj:774]: Why build phases have been reordered?

File: MHIdentityKit/Infrastructure/IdentityManager/IdentityManager.swift [hunk 2]
```diff
@@ -22,7 +22,7 @@ public protocol IdentityManager {
      - parameter handler: The callback, executed when the authorization is complete. The callback takes 2 arguments - an URLRequest and an Error
      */
     
-    func authorize(request: URLRequest, forceAuthenticate: Bool, handler: @escaping (URLRequest, Error?) -> Void)
+    func authorize(request: URLRequest, forceAuthenticate: Bool, skipRefresh: Bool, handler: @escaping (URLRequest, Error?) -> Void)
```
Comment (KoCMoHaBTa) [inline MHIdentityKit/Infrastructure/IdentityManager/IdentityManager.swift:26]: Documentation for `skipRefresh` is missing. This applies to all functions.

File: MHIdentityKit/Infrastructure/UserAgent/macOS/WebViewUserAgentViewController.swift [hunk 3]
```diff
@@ -1,297 +0,0 @@
-//
```
Comment (KoCMoHaBTa) [inline MHIdentityKit/Infrastructure/UserAgent/macOS/WebViewUserAgentViewController.swift]: Why this file is deleted?

**Review by KoCMoHaBTa — CHANGES_REQUESTED** [review]

**Comment by KoCMoHaBTa** [comment]
Also there were some compile issue on macOS, that i've fixed, so please rebase from master, otherwise the CI can't compile.

File: MHIdentityKit.xcodeproj/project.pbxproj (hunk 1 above)
Comment (svetzhelev) [reply MHIdentityKit.xcodeproj/project.pbxproj:774]: With latest xCode order of headers matters, and if not changed on 2nd build a compile error will occur.

File: MHIdentityKit/Infrastructure/UserAgent/macOS/WebViewUserAgentViewController.swift (hunk 3 above)
Comment (svetzhelev) [reply MHIdentityKit/Infrastructure/UserAgent/macOS/WebViewUserAgentViewController.swift]: By mistake probably...

File: MHIdentityKit/Infrastructure/IdentityManager/IdentityManager.swift (hunk 2 above)
Comment (svetzhelev) [reply MHIdentityKit/Infrastructure/IdentityManager/IdentityManager.swift:26]: Done.

File: MHIdentityKit/Infrastructure/UserAgent/macOS/WebViewUserAgentViewController.swift (hunk 3 above)
Comment (svetzhelev) [reply MHIdentityKit/Infrastructure/UserAgent/macOS/WebViewUserAgentViewController.swift]: Fixed.

**Pull request closed without merging**